Bottom: 27bb6ec3459f3aa50e17c80c5b864343f26d7825
Top:    afcefaba8f327e8fc4bd8ff25643f5aaa2b5a99d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-04 12:14:12 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 994f006..7d1a5b8 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1364,8 +1364,8 @@ DrawLayerGroup (int group, const BoxType * screen)
     layernum = layers[i];
     Layer = PCB->Data->Layer + layers[i];
 
-    if (strcasecmp (Layer->Name, "outline") == 0 ||
-        strcasecmp (Layer->Name, "route") == 0)
+    if (strcmp (Layer->Name, "outline") == 0 ||
+        strcmp (Layer->Name, "route") == 0)
       rv = 0;
 
     if (layernum < max_layer /*&& Layer->On*/) {
